lighthouse/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "7.1.0"
 authors = ["Sigma Prime <contact@sigmaprime.io>"]
 edition = { workspace = true }
 autotests = false
-rust-version = "1.87.0"
+rust-version = "1.88.0"
```

Comment:
We don't technically need to bump the MSRV, right?

Reply:
We don't, but we usually bump all the versions together - a bit easier to maintain IMO. 
OR we could keep everything at 1.87, what do you think?